src/performance/memory.ts

```diff
@@ -0,0 +1,21 @@
+import { Event } from "@clarity-types/data";
+import { MemoryData, PerformanceMemory } from "@clarity-types/performance";
+import encode from "./encode";
+
+export let data: MemoryData = null;
+
+export function reset(): void {
+    data = null;
+}
+
+export function compute(): void {
+    if (performance && "memory" in performance) {
```

Comment:
>memory [](start = 24, length = 6)

Add link to spec this is based on. Never heard of it. :-)

Reply:
Done

---
In reply to: [347588259](https://github.com/microsoft/clarity-js/pull/160#discussion_r347588259) [](ancestors = 347588259)